frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestTable.tsx

```diff
@@ -139,43 +140,33 @@ export const ChangeRequestTable: VFC = () => {
             },
             {
                 Header: 'Required approvals',
-                Cell: ({ row: { original } }: any) => {
-                    const { hasAccess } = useContext(AccessContext);
```

Comment:
Is this actually wrong? It's at the top level of a component, right? It's just an anonymous component nested within another one. dosen't seem conditional to me. Still, don't mind changing it if it makes the linter happy 💁🏼